solr/core/src/java/org/apache/solr/update/processor/SkipExistingDocumentsProcessorFactory.java

```diff
@@ -224,6 +224,17 @@ boolean doesDocumentExist(BytesRef indexedDocId) throws IOException {
       }
     }
 
+    boolean doesChildDocumentExist(AddUpdateCommand cmd) throws IOException {
```

Comment:
I'd love to see some javadocs added to these methods.  I know they didn't have it before....

Reply:
I can, although they are effectively private helper methods for internal use by this class, nobody should be calling them from outside. The only reason they're not marked `private` is so I can refer to them in the unit tests.

Does Solr typically put javadoc on non-public methods?

Reply:
hummm...   not sure on the non public!   However, one thing is, I think we have some sort of annotation "VisiblityForTesting" that we I think prefer...   I didn't see the `private` so thought it was public....   Maybe @dsmiley can weigh in on your question.

Reply:
> Does Solr typically put javadoc on non-public methods?

No.  Add if you wish; if it'd be helpful.  Based on what you said above; I wouldn't!
I only use VisibleForTesting on a public method that is public only for testing.  Package scope is kinda whatever -- maybe assumed even.

Reply:
makes sense...